src/components/table/LightTable.tsx

```diff
@@ -289,7 +291,7 @@ function LightTable({
     0,
   );
   const effectiveTableWidth = _.isNaN(tableWidth) || !supportsResize ? '100%' : tableWidth;
-  const showPagination = (rows && rows.length > 0) || infinitePaging;
+  const showPagination = (rows?.length > 0 || infinitePaging) && !hidePagination;
```

Comment:
For the error table, pagination isn't needed so I added a setting to force the pagination controls to be hidden.